dvc/command/metrics.py

```diff
@@ -261,7 +261,7 @@ def add_parser(subparsers, parent_parser):
     metrics_modify_parser.add_argument("path", help="Path to a metric file.")
     metrics_modify_parser.set_defaults(func=CmdMetricsModify)
 
-    METRICS_REMOVE_HELP = "Remove files's metric tag."
+    METRICS_REMOVE_HELP = "Stop tracking a metric file."
```

Comment:
This might be confusing. We are not stopping tracking the file but rather un-marking it as a metric.

Reply:
True. I'm updating this one and `metrics add`, as well as the cmd refs. See f262f483 and iterative/dvc.org/pull/1072. Thanks!